Packages/com.unity.inputsystem/InputSystem/Plugins/XR/GenericXRDevice.cs

```diff
@@ -7,12 +7,33 @@
 
 namespace UnityEngine.InputSystem.XR
 {
-    /// <summary>
```

Comment:
Is this legal to move it inside the class body?
I was looking around and the language spec says they should proceed source code elements, but maybe you know something I don't here?